src/spritecache.cpp

```diff
@@ -661,7 +661,7 @@ bool LoadNextSprite(int load_index, SpriteFile &file, uint file_sprite_id)
 
 	if (type == SpriteType::Invalid) return false;
 
-	if (load_index >= MAX_SPRITES) {
+	if (static_cast<uint>(load_index) >= MAX_SPRITES) {
```

Comment:
shouldn't load_index be a `uint` (or `size_t`?) ?

Reply:
I'll fix this soon